Ginger/GingerCoreNETUnitTest/RunTestslib/CLITest.cs

```diff
@@ -173,8 +192,11 @@ public void CLIGridWithoutParams()
                 CLIProcessor CLI = new CLIProcessor();
 
                 // Act            
-                CLI.ExecuteArgs(new string[] { "grid" });
-
+               
+                Task.Run(async () =>
+                {
+                    await CLI.ExecuteArgs(new string[] { "grid" });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)